ci-automation/release.sh

```diff
@@ -119,6 +122,10 @@ function _inside_mantle() {
       done
     done
     for arch in amd64 arm64; do
+      if [[ "${arch}" == "arm64" ]] && [[ "${CHANNEL}" == "lts" ]]; then
+        continue
+      fi
```

Comment:
```suggestion
```
Plume release must run to publish the AWS AMIs. I don't think we need to disable the Azure run because it only creates an URL.